packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/pack-xdsl-access.controller.js

```diff
@@ -491,21 +495,64 @@ angular.module('managerApp').controller(
         .$promise.then(() => {
           this.hasIncidentOccured = true;
         })
-        .catch((error) => {
-          const errorStatus = get(error, 'status');
+        .catch((err) => {
+          const errorStatus = get(err, 'status');
```

Comment:
Why rename ? `error` was a perfectly fine variable name. Actually I think it's better to avoid abbreviation when possible because the more explicit it is, the better especiallly for maintenance.